html/CarousleCardsPanelDemo.js

```diff
@@ -0,0 +1,257 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import URI from 'urijs'
+
+import { CarousleCardsRow } from '../src/index'
+
+const buildFeaturedExperimentsCards = (host) => {
+  const linkToImage = (imgFileName) => URI(`resources/images/experiment-list-latest/${imgFileName}.png`, host).toString()
```

Comment:
`experiment-list-latest` endpoint has changed to `experiments-summary`. Verify that your images are showing properly in the demo pages.